### PR TITLE
Prune some helm unittests, migrate others to pytest

### PR DESCRIPTION
**Pull request by danielhoherd**
## Description

Prune some helm unittest files that are no longer used.

## Related Issues

Related to:

- https://github.com/astronomer/issues/issues/3716
- https://github.com/astronomer/airflow-chart/pull/263
- https://github.com/astronomer/airflow-chart/pull/268

## Testing

These are deletions that are no longer used in any way. If CI is passing that's testing enough.

## Next steps

This change should be cherry-picked to all supported branches to keep them nice and tidy.

### CODE REVIEW

**Comment by jedcunningham** [comment]
"no longer used" meaning the tests aren't run? Shouldn't we convert them to pytest tests instead?

**Comment by pgvishnuram** [comment]
@danielhoherd - let me add some equivalent pytest

**Comment by pgvishnuram** [comment]
@danielhoherd -lets merge this PR

File: tests/chart_tests/test_scc.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from tests.chart_tests.helm_template_generator import render_chart
+
+from .. import supported_k8s_versions
+
+
+@pytest.mark.parametrize("kube_version", supported_k8s_versions)
+class TestAirflowsccprivileges:
```
Comment (danielhoherd) [inline tests/chart_tests/test_scc.py:9]:
```suggestion
class TestAirflowSccPrivileges:
```

**Comment by danielhoherd** [comment]
> "no longer used" meaning the tests aren't run? Shouldn't we convert them to pytest tests instead?

Yeah, we should. Big oversight on my part. I'll start a new branch to make sure those test cases are covered.

**Pull request merged**